runtime/onert/frontend/circle_traininfo/include/traininfo_loader.h

```diff
@@ -27,7 +27,7 @@ namespace train
 namespace traininfo_loader
 {
 
-static constexpr char *const TRAININFO_METADATA_NAME = "CIRCLE_TRAINING";
+extern const char *const TRAININFO_METADATA_NAME;
```

Comment:
While I was on the next step ([next PR](https://github.com/Samsung/ONE/pull/12408)), I found an error :  
```bash 
ONE/runtime/onert/frontend/circle_traininfo/include/traininfo_loader.h:30:56: 
error: ISO C++ forbids converting a string constant to ‘char*’ [-Werror=write-strings]
   30 | static constexpr char *const TRAININFO_METADATA_NAME = "CIRCLE_TRAINING";
      |                                                        ^~~~~~~~~~~~~~~~~

``` 

It looks like `constexpr char*` doesn't mean that char-array(string) content is constant.
So, For clearness, Let's just remove `constexpr` and use `const` instead.

-------------------------------------------------------------


I refered stack-overflow https://stackoverflow.com/questions/30561104/const-constexpr-char-vs-constexpr-char.
I tried to find an official C++ document about this, But it's hard to find a clear explanation about this.

Reply:
Could you add a comment like below:
```
// TODO Change to inline variables since C++17
```

Reply:
PTAL :)

Reply:
👍